tests/test_url_build.py

```diff
@@ -32,12 +32,16 @@ def test_build_with_scheme_and_host():
     assert u == URL("http://127.0.0.1")
 
 
-def test_build_with_port():
-    with pytest.raises(ValueError):
-        URL.build(port=8000)
-
-    u = URL.build(scheme="http", host="127.0.0.1", port=8000)
-    assert str(u) == "http://127.0.0.1:8000"
+@pytest.mark.parametrize(
+    "port,exc",
```

Comment:
A non-string sequence for the params list is considered more readable, some projects even enforce this with linters:
```suggestion
    ("port", "exc"),
```

Reply:
done